### DISCUSSION TITLE
window-scene/

### DISCUSSION THREAD

**Discussion by giscus** [noah0316/noah0316.github.io#15] [General]
# window-scene/

iOS application에서 화면을 나타내는데 중요한 역할을 하는 window와 scene에 대해 알아봅시다.

scene을 알아보기에 앞서 먼저 window부터 알아보자

window는 애플리케이션 user interface의 전반적인 presentation을 처리한다.

window는 view(및 자체 view controller)와 함께 작동하며 사용자에게 표시되는 view 계층 구조와의 interaction 및 변경사항을 관리한다.

window object는 app의 콘텐츠를 담는 역할을 하며, screen은

https://noah-ios.dev/window-scene/

**Comment by JaydenLee1116** [comment]
잘 보고 갑니다~ :)!

**Comment by hdaisywd** [comment]
도움 많이 되었습니다!

**Comment by Aiden0704** [comment]
너무 맛있네요~ 잘 보고 갑니다! 😁
